core/src/com/po/fuck/Assets.java

```diff
@@ -0,0 +1,30 @@
+package com.po.fuck;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.assets.AssetManager;
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.utils.JsonReader;
+import com.badlogic.gdx.utils.JsonValue;
+
+
+public class Assets {
+    private static JsonReader jsonReader = new JsonReader();
+
+    public static AssetManager manager = new AssetManager();
+
+    public static JsonValue getInfo(String name) {
```

Comment:
`getInfo` - what the info does it return, should be renamed or comment should be added ?